components/elm/src/biogeochem/VOCEmissionMod.F90

```diff
@@ -114,7 +110,7 @@ subroutine InitAllocate(this, bounds)
     ! !LOCAL VARIABLES:
     integer            :: i, imeg
     integer            :: class_num
-    real(r8)           :: factors(numpft)
+    real(r8)           :: factors(numveg)
```

Comment:
is this a bug fix? The allocation space on this changed, but its usage did not.

Reply:
Full range was not used previously.